files/scripts/covid19-etl-job.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 #
-# Runs daily COVID-19 Illinois Department for Public Health
+# Runs daily COVID-19 ETLs
 # Run as cron job in covid19@adminvm user account
 #
 # USER=USER
 # KUBECONFIG=path/to/kubeconfig
-# 0   0   *   *   *    (if [ -f $HOME/cloud-automation/files/scripts/covid19-etl-job.sh ]; then JOB_NAME=<JOB_NAME> bash $HOME/cloud-automation/files/scripts/covid19-etl-job.sh; else echo "no codiv19-etl-job.sh"; fi) > $HOME/covid19-etl-$JOB_NAME-job.log 2>&1
+# 0   0   *   *   *    (if [ -f $HOME/cloud-automation/files/scripts/covid19-etl-job.sh ]; then USER=<USER> S3_BUCKET=<S3_BUCKET> JOB_NAME=<JOB_NAME> bash $HOME/cloud-automation/files/scripts/covid19-etl-job.sh; else echo "no codiv19-etl-job.sh"; fi) > $HOME/covid19-etl-$JOB_NAME-job.log 2>&1
```

Comment:
Can we use `S3_BUCKET` like `USER` on line #6? It would be much easier to not repeat this.

Reply:
oh, i didn't see it on line 6! woops